maas_ml/dash_utils.py

```diff
@@ -286,13 +375,12 @@ def predictions_table(name, version, placeholder, n):
         cols = list()
         path = os.environ['MODELS_PATH'] + "/" + name + "/" + version + "/" + "predictions_data_" + version + ".csv"
         predictions_dataframe = pd.read_csv(path, header=None)
-        predictions_dataframe = predictions_dataframe.iloc[:, :-1]

```

Comment:
@dianajlailaty please don't remove this line of code. I added it to do not display the token id in the predictions preview